unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/SceneController.cs

```diff
@@ -462,8 +460,10 @@ public IParcelScene CreateTestScene(LoadParcelScenesMessage.UnityParcelScene dat
             Environment.i.messaging.manager.AddControllerIfNotExists(this, data.id);
 
             Environment.i.world.state.loadedScenes.Add(data.id, newScene);
+            if (!string.IsNullOrEmpty(entityId))
+                newScene.CreateEntity(entityId);
```

Comment:
This gives this method too many responsibilities. We should create this entity from outside.
```suggestion
                
```